cmd/mock-tikv/main.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/tikv/mock-tikv/server"
+)
+
+func main() {
+	cfg := server.NewConfig()
+	err := cfg.Parse(os.Args[1:])
+
+	defer server.LogPanic()
+
+	switch errors.Cause(err) {
+	case nil:
+	case flag.ErrHelp:
+		os.Exit(0)
+	default:
+		log.Fatalf("parse cmd flags error: %s\n", fmt.Sprintf("%+v", err))
+	}
+
+	err = server.InitLogger(&cfg.Log)
```

Comment:
Do we need to use `pingcap/log` instead of `logrus`? /cc @disksing

Reply:
I think pingcap/log is fine. @rleungx

Reply:
pingcap/log +1

It is faster than logrus.

Reply:
Sure. Store fail points code is almost done. Let me cleanup code and fix these all together.

Reply:
Changed to pingcap/log